src/main/java/io/cdap/plugin/salesforce/authenticator/AuthenticatorCredentials.java

```diff
@@ -97,6 +101,11 @@ public Integer getConnectTimeout() {
     return connectTimeout;
   }
 
+  @Nullable
+  public String getProxyUrl() {
+    return proxyUrl;
```

Comment:
Can you check if the new fields introduced need to be part of the equals and hashcode ?

Reply:
New fields are not required to be part of hashcode or equals method as new fields are non mandatory and we are doing any comparison of AuthenticatorCredentials anywhere or storing it into any hash data structure. We can skip that.

Reply:
Suggestion: Even if they are optional, it would make a difference to the object equality. I agree that it is not relevant to this particular class since it may not be used in hash data structures. However since they are already defined, good to add the new fields to keep it correct.

Reply:
added.